src/qt/bitcoingui.h

```diff
@@ -19,6 +19,9 @@
 #include <QMenu>
 #include <QPoint>
 #include <QSystemTrayIcon>
+#ifdef Q_OS_WIN
+#include <QWinTaskbarButton>
```

Comment:
Use follow declaration instead?

Reply:
Ops, forward declaration.

Reply:
Why?

Reply:
Because you don't need to pull in the entire header file (+ what ever it includes) when using just a pointer. It's preferred in case like this to reduce compile times etc. Include it in CPP instead.

Reply:
☝️also, look in other headers, it's already a recurring practice.

Reply:
Oh, I saw that QT_NAMESPACE. I'll do that after gitian build finished too.